app/controllers/works_controller.rb

```diff
@@ -26,7 +26,7 @@ def create
 
     @form = work_form(work_version)
     if @form.validate(work_params) && @form.save
-      after_save(form: @form, event_context: { user: current_user, description: 'Created' })
+      after_save(form: @form, event_context: build_event_context(@form))
     else
```

Comment:
it turns out we *do* want to show changes for a new work after all, but with some slightly different logic (which will be handled in the event builder class)